src/asset-swapper/utils/market_operation_utils/types.ts

```diff
@@ -332,8 +338,15 @@ export interface PlatypusInfo {
     tokens: string[];
     gasSchedule: number;
 }
+export interface GMXFillDataWithChainId extends FillData {
```

Comment:
nit/optional: seems backward? (feel free to ignore since it's time-sensitive). 

`interface GMXFillDataWithChainId extends GMxFillData`?